app/v1/services/DeleteOtherReliefsService.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.services
+
+import cats.data.EitherT
+import cats.implicits._
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.http.HeaderCarrier
+import utils.Logging
+import v1.connectors.DeleteOtherReliefsConnector
+import v1.controllers.EndpointLogContext
+import v1.models.errors._
+import v1.models.requestData.deleteOtherReliefs.DeleteOtherReliefsRequest
+import v1.support.DesResponseMappingSupport
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class DeleteOtherReliefsService @Inject()(connector: DeleteOtherReliefsConnector) extends DesResponseMappingSupport with Logging {
+
+  def delete(request: DeleteOtherReliefsRequest)(
+            implicit hc: HeaderCarrier,
+            ec: ExecutionContext,
+            logContext: EndpointLogContext): Future[DeleteOtherReliefsServiceOutcome] = {
+    val result = for {
+      desResponseWrapper <- EitherT(connector.deleteOtherReliefs(request)).leftMap(mapDesErrors(desErrorMap))
+    } yield desResponseWrapper
+    result.value
+  }
+
+  private def desErrorMap: Map[String, MtdError] =
```

Comment:
Missing mappings for:
- FORMAT_STATUS
- CLIENT_OR_AGENT_NOT_AUTHORISED (this one probably won't be in the mapping once we do alignments but it's best to make these as true to the spec as-is as we can and just align it all later)